yarn.lock

```diff
@@ -12992,17 +12980,6 @@ markdown-it@^12.2.0, markdown-it@^12.3.0:
     mdurl "^1.0.1"
     uc.micro "^1.0.5"
 
-markdown-it@^12.3.0:
```

Comment:
only thing changing in the lock file should be the component-library.  Something's out of sync here. I'd take the latest lock from develop and run a fresh install.

Reply:
nvm i think you can disregard this comment. Pretty sure this is just a side-effect of cleaning up duplicates that for some reason exist in the yarn.lock in develop right now.